### ISSUE TITLE
add identifier to outputs

### ISSUE THREAD

**Issue by shilikok** [terraform-aws-modules/terraform-aws-rds#495]
## Is your request related to a new offering from AWS?
Is this functionality available in the [AWS provider for Terraform](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)? See [CHANGELOG.md](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md), too.
  - Yes ✅: please list the AWS provider version which introduced this functionality
aws provider: 5.0.0

## Is your request related to a problem? Please describe.
id is no longer the AWS database identifier - id is now the dbi-resource-id. Refer to identifier instead of id to use the database's identifier

## Describe the solution you'd like.
add to module outputs db identifier value

**Comment by bryantbiggs** [comment]
while the attribute may have always been there, the value for `id` has changed between < v4.x & > v5.x 

so while simply adding the `identifier` output seems harmless, I don't think that is providing value without addressing this underlying change

**Comment by daroga0002** [comment]
issue is that database name is not returned on v5 provider by any parameter. After adding this value it module will become working correctly with v5 provider

**Comment by daroga0002** [comment]
@shilikok  FYI

I am currently adressing this by making workaround by extracting from instance ARN last element which represent identifier of database

```hcl
module "database_replica_rds" {
  count   = local.create_database_replica_rds ? 1 : 0
  source  = "terraform-aws-modules/rds/aws"
  version = "5.9.0"

  # Source database. For cross-region use db_instance_arn
  replicate_source_db        = element(split(":", module.database_rds[0].db_instance_arn), length(split(":", module.database_rds[0].db_instance_arn))-1)
  .....
}
```

**Comment by AyodejiO** [comment]
I just had the same issue where I had to manually use the `local.identifier` variable used to create the master DB. You should prolly use that to remove the complex code you have @daroga0002

**Comment by daroga0002** [comment]
> I just had the same issue where I had to assign the `replicate_source_db` to the `local.identifier` variable used to create the master DB. You should prolly use that to remove the complex code you have @daroga0002

It will be not working when you use [instance_use_identifier_prefix](https://github.com/terraform-aws-modules/terraform-aws-rds#input_instance_use_identifier_prefix) variable.

What is more it will require additional manual dependency establish between main and replica module (when you use outputs from one module dependency is done automaticlly)

**Comment by daroga0002** [comment]
@bryantbiggs @antonbabenko  
I really dont understand what really problem is to add this output to module. It will not brake any compatibility with any currenrtly supported terraform and provider versions. Instead it will allow to use aws provider v5.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 6.0.0 :tada:
